inbox/core/app_push/backends/firebase.py

```diff
@@ -21,8 +21,15 @@ def __init__(self, fail_silently=False, dry_run=False):
 
         self.dry_run = dry_run
 
+        settings = inbox_settings.get_config()["BACKENDS"]["APP_PUSH_CONFIG"]
+        service_account_file = settings.get("SERVICE_ACCOUNT_FILE")
+        credentials = settings.get("CREDENTIALS")
+        project_id = settings.get("PROJECT_ID")
+
         self.fcm = FCMNotification(
-            api_key=inbox_settings.get_config()['BACKENDS']['APP_PUSH_CONFIG']['GOOGLE_FCM_SERVER_KEY']
+            service_account_file=service_account_file,
+            credentials=credentials,
```

Comment:
I'm assuming we'll use ADC (credentials) and not service_account_file and this is just for completeness since it's an open source library we have...

Reply:
Yeah you'd just just credentials and service_account_file to None